src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
+      position: relative;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      .nav__link {
+        display: flex;
+        align-items: center;
+        height: 60px;
+        text-decoration: none;
+        color: $main-color;
+        font-size: 12px;
+        font-weight: 500;
+        text-transform: uppercase;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), font sizes should be defined in pixels. Consider changing the `font-size` unit from `px` to `rem` to maintain scalability and accessibility.